server/audit_math/suite.py

```diff
@@ -631,20 +619,22 @@ def get_sample_size(
             or (expected_pvalue is np.nan)
         ):
             mid_n = int(np.floor((low_n + high_n) / 2))  # cast for typechecker
-            if mid_n in [low_n, high_n]:
-                break
             mid_pvalue = try_n(
                 mid_n, alpha, contest, winner, loser, bp_stratum, cvr_stratum, n_ratio,
             )
+            if mid_n in [low_n, high_n]:
+                break
```

Comment:
why this change?

Reply:
I was debugging and needed to see the value before it broke. I put it back!